application/package/rpm/MultiMC5.spec

```diff
@@ -0,0 +1,45 @@
+Name:           MultiMC5
+Version:        1.4
+Release:        1%{?dist}
+Summary:        A local install wrapper for MultiMC
+
+License:        ASL 2.0
+URL:            https://multimc.org
+#Source0:        
+BuildArch:      x86_64
+
+#BuildRequires:  
+Requires:       zenity qt5-qtbase wget
+Provides:       multimc MultiMC multimc5
+
+%description
+A local install wrapper for MultiMC
+
+%prep
+#%%autosetup
+
+
+%build
+
+
+%install
+mkdir -p %{buildroot}/opt/multimc
+install -m 0644 ../ubuntu/multimc/opt/multimc/icon.svg %{buildroot}/opt/multimc/icon.svg
+install -m 0755 ../ubuntu/multimc/opt/multimc/run.sh %{buildroot}/opt/multimc/run.sh
+mkdir -p %{buildroot}/%{_datadir}/applications
+install -m 0644 ../ubuntu/multimc/usr/share/applications/multimc.desktop %{buildroot}/%{_datadir}/applications/multimc.desktop
+
+
+%files
+#%%license add-license-file-here
+#%%doc add-docs-here
```

Comment:
I think it would be better to remove unused lines entirely. The spec file parser is puree jank and could still be looking at this.